filtering/filter_computers.py

```diff
@@ -352,10 +375,12 @@ def reservable(  # noqa: C901
                     final_choices[pick] = [choice, available[pick][choice]["name"]]
                     del sorted_available[pick]
                     break
-    return final_choices
+    return available, final_choices
```

Comment:
The docstring for `reservable()` could be updated to reflect that we are now returning a tuple rather than a single dictionary